### DISCUSSION TITLE
v3.5.0

### DISCUSSION THREAD

**Discussion by mattieserver** [mattieserver/netbox-topology-views#289] [Announcements]
## What's Changed
* Fixes #285: Remove 'Icons info' from Readme by @dreng in https://github.com/mattieserver/netbox-topology-views/pull/287
* Closes #189, Closes #158: Add draw.io XML export by @dreng in https://github.com/mattieserver/netbox-topology-views/pull/277
* Closes #64: Show neighbors option by @dreng in https://github.com/mattieserver/netbox-topology-views/pull/286
* Prepare release v3.5.0 by @mattieserver in https://github.com/mattieserver/netbox-topology-views/pull/288

**Full Changelog**: https://github.com/mattieserver/netbox-topology-views/compare/v3.4.0...v3.5.0

<hr /><em>This discussion was created from the release <a href='https://github.com/mattieserver/netbox-topology-views/releases/tag/v3.5.0'>v3.5.0</a>.</em>

**Comment by pablosantosubt** [comment]
Olá, bom dia, eu estou utilizando a versão 3.4.10 do Netbox, eu instalei a versão 3.5.0 do pluguin, mas em meu netbox em topologia não aparece a topologia da minha rede conectada
![image](https://github.com/mattieserver/netbox-topology-views/assets/102676921/009c2f07-8eba-444b-a9cd-359b7568dc70)
![image](https://github.com/mattieserver/netbox-topology-views/assets/102676921/8c2f6d31-19a0-4be9-8925-bf3862e45d92)


poderiam me ajudar por favor?

**Comment by dreng** [comment]
I'm not sure if anyone speaks spanish here, but it would help for sure to switch to english.

First thing I can tell is that you didn't read the manual. Otherwise your config wouldn't look like this. You didn't set any filter options. That might be your problem, whatever your problem is.

**Reply by pablosantosubt** [reply]
Hello, thanks for the reply, I'm Brazilian, I apologize for writing in Portuguese, but I analyzed the entire installation and managed to correct the problem, then I did the path and it worked. thanks!

cd /opt/netbox/netbox
python3 manage.py migrate netbox_topology_views
python3 manage.py collectstatic --no-input

**Comment by dreng** [comment]
Great you worked it out. Apologies for thinking it was Spanish. I didn't mean to be offensive.
